packages/http/src/decorators.ts

```diff
@@ -104,9 +104,7 @@ export function $query(
       }
       const format = queryNameOrOptions.properties.get("format")?.type;
       if (format?.kind === "String") {
-        if (format.value === "multi" || format.value === "csv") {
-          options.format = format.value;
-        }
+        options.format = format.value as any; // That value should have already been validated by the TypeSpec dec
```

Comment:
When I try to test the effect of extern dec in the playground I get this:
![image](https://user-images.githubusercontent.com/5723682/232080038-e6ea3fc1-4d70-491e-b147-f1890d91b3a7.png)

Not sure if I'm just doing it wrong.

Reply:
Turns out it validates if you used a good value, but the error you get when using a bad one is less than ideal. It would be better if it said "format expected to be: ...." but this isn't technically wrong... because the anonymous model as written isn't assignable to QueryOptions.

Reply:
Can we create a tracking issue to see if anything can be done about the error in this case?